### DISCUSSION TITLE
Attempting to call a managed function

### DISCUSSION THREAD

**Discussion by germj** [nxrighthere/UnrealCLR#144] [Help] [answered]
Hi,

I've downloaded the latest version of the plugin. And it's changed from the last time I've used it. 

How do you call a managed function from an empty new level? 

![image](https://user-images.githubusercontent.com/58347637/120089402-e51f7100-c0c7-11eb-99ad-bcbd33e9d09e.png)

Any support will be greatly appreciated.

**Comment by nxrighthere** [comment, marked as answer]
You can find [examples in tests](https://github.com/nxrighthere/UnrealCLR/blob/master/Source/Managed/Tests/BlueprintsExtensibility.cs#L332). Check [blueprint actor](https://github.com/nxrighthere/UnrealCLR/blob/master/Content/Tests/BlueprintActor.uasset) or [blueprint scene component](https://github.com/nxrighthere/UnrealCLR/blob/master/Content/Tests/BlueprintSceneComponent.uasset).

**Comment by nxrighthere** [comment]
Did you manage it?
